README.md

```diff
@@ -1,100 +1,28 @@
-    [OpenCV](https://software.llnl.gov/lbann/opencv.html)
-    CUDA (optional)
-    cuDNN (optional)
-    [Protocol Buffers](https://software.llnl.gov/lbann/protobuf.html) (optional)
-    [Doxygen](https://software.llnl.gov/lbann/doxygen.html) (optional)
-    *Note: LBANN also requires a C++ compiler with OpenMP support. The GCC 5.0 and Intel 16.0 C++ compilers are recommended*
-    2. Clone this repo using `git clone https://github.com/LLNL/lbann.git`
-    3. In the main LBANN directory create a build directory using `mkdir build`
-    4. `cd` into this directory and run the following commands
-    `cmake ../..`
-    `make`
-    `make install`
-    *Note: It may be necessary to manually set CMake variables to control the build configuration*
-
-## Verifying LBANN on LC
-   1. Allocate compute resources using SLURM: `salloc -N1 -t 60`
-   2. Run a test experiment for the MNIST data set; from the main lbann directory run the following command:
- ```
-  srun -n12 build/gnu.catalyst.llnl.gov/install/bin/lbann \
---model=model_zoo/models/lenet_mnist/model_lenet_mnist.prototext \
---reader=model_zoo/data_readers/data_reader_mnist.prototext \
---optimizer=model_zoo/optimizers/opt_adagrad.prototext \
---num_epochs=5
-```
-Note: `srun -n12 build/gnu.catalyst.llnl.gov/install/bin/lbann` assumes you are running on the LLNL catalyst platform;
-  if running on some other platform, and/or have installed lbann in a different directory, you
-  will need to adjust this command.
-
-  This should produce roughly the following final results on Catalyst:
-```
---------------------------------------------------------------------------------
-[4] Epoch : stats formated [tr/v/te] iter/epoch = [844/94/157]
-            global MB = [  64/  64/  64] global last MB = [  48  /  48  /  16  ]
-             local MB = [  64/  64/  64]  local last MB = [  48+0/  48+0/  16+0]
---------------------------------------------------------------------------------
-Model 0 training epoch 4 objective function : 0.0471567
-Model 0 training epoch 4 categorical accuracy : 99.6241%
-Model 0 training epoch 4 run time : 7.64182s
-Model 0 training epoch 4 mini-batch time statistics : 0.00901458s mean, 0.0212693s max, 0.0078979s min, 0.000458463s stdev
-Model 0 validation objective function : 0.0670221
-Model 0 validation categorical accuracy : 98.9%
-Model 0 validation run time : 0.25341s
-Model 0 validation mini-batch time statistics : 0.00269454s mean, 0.00285273s max, 0.0020936s min, 6.65695e-05s stdev
-Model 0 test objective function : 0.0600125
-Model 0 test categorical accuracy : 99.02%
-Model 0 test run time : 0.421912s
-Model 0 test mini-batch time statistics : 0.00268631s mean, 0.00278771s max, 0.00131827s min, 0.00011085s stdev
-``` 
-  Note: LBANN performance will vary on a machine to machine basis. Results will also vary, but should not do so significantly. 
+A few options for building LBANN are documented [here](docs/BuildingLBANN.md#top).
```

Comment:
You should explicitly mention how a _user_ should install LBANN here, with a pointer to further details and developer documentation.

Reply:
@bvanessen How's `spack install lbann` coming?